sdk/core/System.Net.ClientModel/README.md

```diff
@@ -0,0 +1,41 @@
+# System.Net.ClientModel shared client library for .NET
+
+`System.Net.ClientModel` provides shared primitives, abstractions, and helpers for .NET service client libraries.
+
+`System.Net.ClientModel` allows client libraries built from its components to expose common functionality in a consistent fashion, so that once you learn how to use these APIs in one client library, you'll know how to use them in other client libraries as well.
+
+## Getting started
+
+Typically, you will not need to install `System.Net.ClientModel`.
+it will be installed for you when you install one of the client libraries using it.
+
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/).
+
+### Prerequisites
+
+None needed for `System.Net.ClientModel`.
+
+## Key concepts
+
+The main shared concepts of `System.Net.ClientModel` include:
+
+- Providing APIs to read and write models in different formats.
+
+## Examples
```

Comment:
We minimally need to add examples for any types we want to put in the root namespace, as these are end-user consumer types that callers should be able to use on day 1.

If we decide to move types out of the root namespace and not provide any examples when we first GA the library, we should remove this section from the README so we don't have empty sections.